src/assessment.py

```diff
@@ -58,17 +60,22 @@ def post_assessment():
     except KeyConceptError as e:
         return e, 400
     except OpenaiServerError as e:
-        return e, 503
+        return f"OpenAI server error: #{e}: ", 503
+    except BedrockServerError as e:
+        return f"Bedrock server error: #{e}: ", 503
     except requests.exceptions.ReadTimeout as e:
-        return f"OpenAI timeout: #{e}: ", 504
+        if 'gpt' in llm_model:
+            return f"OpenAI timeout: #{e}: ", 504
+        elif 'bedrock' in llm_model:
+            return f"Bedrock timeout: #{e}: ", 504
```

Comment:
I don't think we want to silently drop this error in other cases, for example if we try out another LLM (or if we switch to accessing AWS models without bedrock). I would suggest adding a final else clause:
```
        else:
            return f"LLM timeout: #{e}: ", 504
```